logging/logging_test.go

```diff
@@ -14,6 +14,10 @@ func TestRoundTrip(t *testing.T) {
 		test.That(t, err, test.ShouldBeNil)
 		test.That(t, parsed, test.ShouldEqual, level)
 	}
+
+	parsed, err := LevelFromString("warning")
+	test.That(t, err, test.ShouldBeNil)
+	test.That(t, parsed, test.ShouldEqual, WARN)
```

Comment:
I am a little hesitant on where to add this test, since `TestRoundTrip` tests the strings for pretty much every other level. I wasn't sure if it was a big enough test to add another whole test, or to add it like this. Maybe if I rename the test to `TestLevelStrings`?